sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Blinding.scala

```diff
@@ -83,23 +83,24 @@ object Blinding {
 
   private[engine] def partyPackages(
       tx: VersionedTransaction,
-      blindingInfo: BlindingInfo,
+      disclosure: Relation[NodeId, Party],
+      contractVisibility: Relation[ContractId, Party],
       contractPackages: Map[ContractId, Ref.PackageId],
   ): Relation[Party, Ref.PackageId] = {
     Relation.from(
-      disclosedPartyPackages(tx, blindingInfo.disclosure) ++
-        divulgedPartyPackages(contractPackages, blindingInfo.divulgence)
+      disclosedPartyPackages(tx, disclosure) ++
+        contractPartyPackages(contractPackages, contractVisibility)
     )
   }
 
   // These are the packages needed for model conformance
```

Comment:
You mean needed for "input contract validation"? Model conformance needs other packages, not included here.